### PR TITLE
Remove unnecessary context variable and rework Conn structure

### PR DESCRIPTION
**Pull request by sourque**
Should make functions easier to test

### CODE REVIEW

File: src/utility.go [hunk 1]
```diff
@@ -6,9 +6,13 @@ import (
 	"time"
 )
 
+var aeacusVersion = "1.2.0"
```
Comment (safinsingh) [inline src/utility.go]: 1.3.0?

**Review by safinsingh — COMMENTED** [review]
Looks pretty good to me so far

File: TODO.md [hunk 2]
```diff
@@ -38,6 +39,9 @@
   - linux
     - clear ff history?
 
+- cool/stretch goal
```
Comment (safinsingh) [inline TODO.md:42]: We could make `data-binary` on `.binary` initially their ID

Comment (sourque) [reply TODO.md:42]: The `mc.TeamID` field is refreshed every cycle, so we could just add it to `genReport()` I think.

**Review by safinsingh — APPROVED** [review]
lgtm

**Pull request merged**